plugins/ci/skills/payload-snapshot/scripts/payload_snapshot.py

```
@@ -748,18 +775,39 @@ def collect(self) -> bool:
 
+        errors_before = _collection_error_count()
         output = _run_gcloud(
             ["gcloud", "storage", "ls", f"{base}/artifacts/**/junit*.xml"],
-            timeout=30,
+            timeout=120,
         )
-        if not output:
-            return []
 
-        files = [l.strip() for l in output.strip().splitlines() if l.strip()]
+        files = [l.strip() for l in (output or "").strip().splitlines()
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)